packages/cardano-services/src/StakePool/DbSyncStakePoolProvider/mappers.ts

```diff
@@ -167,8 +167,8 @@ export const toStakePoolResults = (
 };
 
 export const mapPoolUpdate = (poolUpdateModel: PoolUpdateModel): PoolUpdate => ({
-  id: poolUpdateModel.id,
-  updateId: poolUpdateModel.update_id
+  id: Number(poolUpdateModel.id),
```

Comment:
If this is `bigint` in the db, then `9223372036854775807 > Number.MAX_SAFE_INTEGER`. Although not likely to be an issue if those are incremental ID fields.

```suggestion
  id: BigInt(poolUpdateModel.id),
```

Reply:
Yep, those are incremental ones.

Reply:
Currently on average ~5000 stake pools, with fewer more updates each, it shouldn't be an issue the incremental id to reach `Number.MAX_SAFE_INTEGER`.